cmd/ftl/cmd_serve.go

```diff
@@ -91,6 +106,77 @@ func (s *serveCmd) Run(ctx context.Context) error {
 	return nil
 }
 
+func runInBackground(ctx context.Context, logger *log.Logger) {
+	pidFilePath := pidPath(logger)
+	if _, err := os.Stat(pidFilePath); err == nil {
```

Comment:
You'll need to check if the process is running too. Load the PID and use `os.Kill(pid, 0)` to do this.